packages/core/src/artifacts.ts

```diff
@@ -79,15 +79,25 @@ export function getBuiltKeystoneConfigurationPath(cwd: string) {
 }
 
 export function getSystemPaths(cwd: string, config: KeystoneConfig) {
+  const builtTypesPath = config.types?.path
+    ? path.join(cwd, config.types.path)
+    : 'node_modules/.keystone/types.d.ts';
+
+  const relativePrismaPath = config.db.prismaClientPath
+    ? `./${path.relative(path.dirname(builtTypesPath), config.db.prismaClientPath)}`
```

Comment:
@dcousens is Windows going to be happy with this?

Reply:
I don't know

Reply:
It's the same problem though, we probably need to `JSON.stringify` it at the least

Reply:
Although the `JSON.stringify` in the `schema-type-printer` should excape the string

Reply:
Actually yes, this string replaced what was passed previously